analysis_module/utils.py

```diff
@@ -150,3 +154,41 @@ def send_callback_url_request(callback_url: str, client_id: str, filepath: str,
 
     logging.error("No callback url found.")
     return json.dumps({"status": "No callback url found."}), 400
+
+
+def send_ecs_http_request(nlp_request: NLPRequest):
+    ecs_id_param_name = get_ecs_id_param_name(nlp_request.type)
+    url = get_ecs_url(nlp_request.type)
+    if url is None:
```

Comment:
Shouldn't we check `ecs_id_param_name` as well?

Reply:
No, not here. It is checked below that part.